docs/website/content/docs/v0.1/Getting Started/concepts.md

```diff
@@ -12,17 +12,25 @@ The Metal Controller Manager provides a few custom resources (CRDs) in the manag
 These define a desired deployment environment for Talos.
 This includes things like which kernel to use, kernel args to pass, and the initrd to use.
 
+See the [Environments](/docs/v0.1/Configuration/environments.md) section of our Configuration docs for examples and more detail.
+
 ## Servers
 
 These represent physical machines as resources in the management plane.
 These servers are created when the physical machine PXE boots and completes a "discovery" process in which it registers with the management plane and provides SMBIOS information.

```

Comment:
"provides SMBIOS information such as the CPU manufacturer and version, and memory information."